src/app/hearing-part/components/hearings-filter/hearings-filter.component.ts

```diff
@@ -0,0 +1,47 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { Judge } from '../../../judges/models/judge.model';
+import { DEFAULT_HEARING_FILTERS, HearingsFilters } from '../../models/hearings-filter.model';
+import { CaseType } from '../../../core/reference/models/case-type';
+import { HearingType } from '../../../core/reference/models/hearing-type';
+import { Priority } from '../../models/priority-model';
+import { ListingStatus } from '../../models/listing-status-model';
+
+@Component({
+    selector: 'app-hearings-filter',
+    templateUrl: './hearings-filter.component.html',
+    styleUrls: ['./hearings-filter.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class HearingsFilterComponent {
+    @Output() onFilter = new EventEmitter();
+
+    @Input() judges: Judge[];
+    @Input() caseTypes: CaseType[];
+    @Input() hearingTypes: HearingType[];
+
+    caseTitleMaxLength = 200;
+    caseNumberMaxLength = 200;
+    priorities = Object.keys(Priority);
+    listingStatuses = {
+        all: {
+            value: ListingStatus.All,
+            label: 'All'
+        } ,
+        listed: {
+            value: ListingStatus.Listed,
+            label: 'Listed'
+        } ,
+        unlisted: {
+            value: ListingStatus.Unlisted,
+            label: 'Unlisted'
+        } ,
+    };
+    communicationFacilitators = ['Sign Language', 'Interpreter', 'Digital Assistance', 'Custom'];
```

Comment:
these values are set already in **listing-create.component.ts**. Please create enum from it and use 
```js
communicationFacilitators  = Object.values(NewEnum)
```